substrate/frame/system/src/lib.rs

```diff
@@ -955,23 +955,16 @@ pub mod pallet {
 
 	/// Events deposited for the current block.
 	///
-	/// NOTE: The item is unbound and should therefore never be read on chain.
+	/// NOTE: The map is unbound and should therefore never be iterated on chain.
 	/// It could otherwise inflate the PoV size of a block.
 	///
 	/// Events have a large in-memory size. Box the events to not go out-of-memory
 	/// just in case someone still reads them from within the runtime.
 	#[pallet::storage]
-	#[pallet::whitelist_storage]
 	#[pallet::disable_try_decode_storage]
```

Comment:
I’m not sure I understand the docs here, they could either mean:
- we don’t need to migrate due to this attribute, or
- we need to remove this attribute in order to migrate

https://paritytech.github.io/polkadot-sdk/master/frame_support/pallet_macros/attr.disable_try_decode_storage.html